akka-javasdk/src/main/java/akka/javasdk/eventsourcedentity/ReplicationFilter.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (C) 2021-2024 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.javasdk.eventsourcedentity;
+
+import akka.javasdk.impl.effect.ReplicationFilterImpl;
+
+public interface ReplicationFilter {
```

Comment:
I want to have this rather open for future additions of other types of filters, than regions.